Makefile

```diff
@@ -207,14 +181,23 @@ Dockerfile.exporter: ./build/ndm-exporter/Dockerfile.in
 build.ndm:
 	@echo '--> Building node-disk-manager binary...'
 	@pwd
+	# CTLNAME=ndm BUILDPATH=ndm_daemonset sh -c "'$(PWD)/build/build.sh'"
 	@CTLNAME=${NODE_DISK_MANAGER} BUILDPATH=${BUILD_PATH_NDM} sh -c "'$(PWD)/build/build.sh'"
 	@echo '--> Built binary.'
 	@echo
 
-.PHONY: docker.ndm
-docker.ndm: build.ndm Dockerfile.ndm 
-	@echo "--> Building docker image for ndm-daemonset..."
-	@sudo docker build -t "$(DOCKER_IMAGE_NDM)" ${DBUILD_ARGS} -f Dockerfile.ndm .
+# Build ndm docker image with buildx
+# Experimental docker feature to build cross platform multi-architecture docker images
+# https://docs.docker.com/buildx/working-with-buildx/
+.PHONY: docker.buildx.ndm
+docker.buildx.ndm:
+	export DOCKER_CLI_EXPERIMENTAL=enabled
```

Comment:
Does an `export` inside the target work?

Reply:
Yes it does 👍